cfme/base/ui.py

```diff
@@ -1709,12 +1709,22 @@ class import_git(View):     # noqa
     reset_all = VersionPick({
         Version.lowest(): Image(
             './/img[starts-with(@alt, "Reset all components in the following domains:")]'),
-        '5.10': Button(title='Reset all components in the following domains:')
+        '5.10': Button(title='Reset all components in the following domains: RedHat, ManageIQ')
     })
+    reset_title = Text(
+        ".//div[contains(@class, 'import-or-export')]/h3[contains(text(), 'Reset all')]"
+    )
 
     @property
     def is_displayed(self):
-        return self.in_import_export and self.export_all.is_displayed
+        # TODO: Remove check if the fix for BZ 1686054 is back ported to 5.9z
+        if self.appliance.version < "5.10":
```

Comment:
PRT highlights the bug here - you're in a View class, which doesn't have an appliance. You can use `self.browser.product_version`